File: MercaderSG/Seguridad/PantallaError/ResultadoFamPatUsu.vb
Public Class ResultadoFamPatUsu

    Private Sub ErrorFamPatUsu_Load(sender As System.Object, e As System.EventArgs) Handles MyBase.Load
        Me.Text = My.Resources.ArchivoIdioma.InformeErrores
    End Sub

    Private Sub ResultadoFamPatUsu_KeyDown(sender As System.Object, e As System.Windows.Forms.KeyEventArgs) Handles MyBase.KeyDown
        If e.KeyCode = Keys.Escape Then
            Me.Close()
        End If
    End Sub
End Class

File: MercaderSG/Seguridad/PantallaError/ResultadoFamPatUsu.cs
using System;
using System.Windows.Forms;

namespace MercaderSG
{
    public partial class ResultadoFamPatUsu
    {
        public ResultadoFamPatUsu()
        {
            InitializeComponent();
        }

        private void ErrorFamPatUsu_Load(object sender, EventArgs e)
        {
            Text = My.Resources.ArchivoIdioma.InformeErrores;
        }

        private void ResultadoFamPatUsu_KeyDown(object sender, KeyEventArgs e)
        {
            if (e.KeyCode == Keys.Escape)
            {
                Close();
            }
        }
    }
}